govc/cluster/override/change.go

```diff
@@ -53,9 +53,11 @@ func (cmd *change) Register(ctx context.Context, f *flag.FlagSet) {
 	// HA
 	rp := []string{
 		string(types.DasVmPriorityDisabled),
```

Comment:
Based on the API ref link you posted, we should be using `types.ClusterDasVmSettingsRestartPriority*` for all the options in this list.  Values are the same, but see comment below on the generated code.